README.md

```diff
@@ -1,79 +1,60 @@
-
-Once you are comfortable with the getting started steps, feel free to explore building out the app using the rest of the code in this repo.
-
-There is also a getting started video if that's easier: [https://www.youtube.com/watch?v=ZEH9kOwDgx0](https://www.youtube.com/watch?v=ZEH9kOwDgx0). 
-
 ### Clone or download this repo
 
 ```bash
   git clone https://github.com/andygup/angular-cli-esri-map.git
 ```
 
-### Install Angular CLI and generate a new project
-
-Create a basic Angular CLI project. Make sure you have Angular CLI installed using standard CSS. See the instructions here: https://cli.angular.io/: 
-
-### Install `esri-loader` and the Esri TypeScript types
-
-You need [`esri-loader`](https://github.com/Esri/esri-loader#usage). It is a low level service that helps load ArcGIS JavaScript API modules (v3.x or v4.x) in non-Dojo applications.
-
-And, the ArcGIS JavaScript TypeScript type definitions can be found [here](https://github.com/Esri/jsapi-resources/tree/master/4.x/typescript). 
-Note: using the Esri Types is optional. However, you'll need to modify this repos code to work without Types.
+### Install dependencies
 
 ```bash
-  npm install --save esri-loader
-  npm install --save @types/arcgis-js-api
+  cd angular-cli-esri-map
+  npm install
 ```
 
-### Generate the scaffolding for your mapping component
+## Working with Angular CLI
 
-```bash
-  ng generate component esri-map
+### Run the application locally
 
-  # or you can also use shorthand
-  ng g component esri-map
+```bash
+  ng serve
 ```
 
-Copy the contents from the `angular-cli-esri-map/src/app/esri-map` folder into your new `esri-app/src/app/esri-map` directory.
+Navigate to http://localhost:4200 to see the app running. The app will automatically reload if you change any of the source files. You can shut down the development server with a `Control C` any time you wish.
 
-Copy the `app.components` contents from `angular-cli-esri-map/src/app` into your new `esri-app/src/app/` directory.
+### Code scaffolding
 
-If you are using a version less than Angular 7, the following steps maybe required if you want to use the Esri Typescript Types.
+Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
-* In `tsconfig.app.json` add `"types": ["arcgis-js-api"]`. 
+### Build
 
-* In `tsconfig.spec.json` add `"types": ["arcgis-js-api"]`. 
+Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
 
-### Make sure the mapping app runs
+### Running unit tests
 
-Now run the following command and you should see our mapping appear on the page.
-
-```bash
-  ng serve
-```
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
```

Comment:
I haven't looked yet, did we add any basic tests?

Reply:
Yes, the original repo had basic unit tests and I doublechecked that they pass in the new clie project